README.md

```diff
@@ -102,3 +102,11 @@ curl --location --request POST 'https://rds-mph.swissartresearch.net/rest/reconc
 ```bash
 curl --location --request POST 'https://rds-mph.swissartresearch.net/rest/reconciliation' --header 'Content-Type: application/json' --header 'Accept: application/json' --data-urlencode 'queries={"q2":{"query":"Car","limit":3},"q3":{"query":"Home","limit":3},"q0":{"query":"Meinheim","limit":3}}'
 ```
+
+## How to start you own RDS-Local instance
+First of all you should ensure that [docker](https://www.docker.com) and [docker-compose](https://docs.docker.com/compose/install/) are installed on your computer. Once this step is done, you can move to the second one.
+Navigate to `./docker-compose-configuration` folder and execute the script `./start.sh`. This script will up the platform in a few minutes, and then you can access the platform using following url: [http://localhost:10214](http://localhost:10214).
+
+After the platform have been up, local blazegraph journal will be mounted to `./docker-compose-configuration/rds-local-blazegraph-journal` folder.
```

Comment:
`When the platform is up, the local blazegraph journal is mounted from the ...`

Reply:
done.